the-collab-lab/tcl-17-smart-shopping-list
# Bl add icons

## Description
This PR will add icons to help make the user experience more enjoyable, create a clean / fun look and feel and simplify the design. This PR will also make some updates to the styling of the navigation to update the UI. 

## Related Issue
closes [#42](https://github.com/the-collab-lab/tcl-17-smart-shopping-list/issues/42)

## Acceptance Criteria

- [ ] Icons should be understood universally
- [ ] The purpose and/or intent of the icons should be clear
- [ ] Use icons minimally to keep the look of the app clean
- [ ] Icons should not make the application harder to use
- [ ] Add font awesome shopping cart, trash can, list icons
- [ ] Shopping bag from flaticon

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |

## Updates

### Before
<img width="416" alt="before-icons" src="https://user-images.githubusercontent.com/36529777/109103226-545b8280-76df-11eb-9ee5-17e9d4c362a7.PNG">


### After
<img width="459" alt="after-icons" src="https://user-images.githubusercontent.com/36529777/109103209-4d347480-76df-11eb-9165-2be8dfcb1b9e.PNG">


## Testing Steps / QA Criteria
* One way to test the site is to click on the `details` button below to open the `Netlify preview`

![Screen Shot 2021-01-12 at 6 05 51 PM](https://user-images.githubusercontent.com/14856004/104389400-ff81f500-5500-11eb-95ae-4c0169a069cb.png)

* Otherwise from your terminal, pull down this branch with `git pull origin BL-add-icons` and check that branch out with `git checkout BL-add-icons`
* `npm start` to launch the app.
* confirm that you see the following icons
1.  shopping cart icon with plus sign for the `Add Item` nav link
2.  list icon for the `Shopping List` nav link
3.  shopping bag svg for the `Home` nav link
4. trash can icon for each list item
